### ISSUE TITLE
[Bug]-[42000]:Welcome tutorial 5th Step, binding image to table data errors out as the backend seems to have changed.

### ISSUE THREAD

**Issue by btsgh** [appsmithorg/appsmith#15118]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Signup as a new user, login and go through the Welcome Tutorial - reach the step 5 of 9, and try to bind the image widget with - {{CustomersTable.selectedRow.image}}, an error pops up - "This value does not evaluate to type: base64 encoded image | data uri | image url"

### Steps To Reproduce

1. Go through the welcome tutorial and reach step 5 of 9 
2. On binding the image to {{CustomersTable.selectedRow.image}}, an error pops up

This value does not evaluate to type: base64 encoded image | data uri | image url

On checking the table for image column value, there are no images there, and just a text T1C. 

Looks like the database has got changed/updated. For welcome tutorial, this is not a great experience for the user. 

### Version

Production

**Comment by kocharrahul7** [comment]
## Stats
| Stat | Values |
| ------------- | ------------- |
| Reach     | 7000 |
| Effort (months)  | 0.5 |

**Comment by akash-codemonk** [comment]
The mock users db does not seem to have a valid image url. The db needs to be reset to default data. 
@kocharrahul7 @sum35h

**Comment by sum35h** [comment]
@akash-codemonk  could you use the `users` table in the `fakeapi` db.

**Comment by Nikhil-Nandagopal** [comment]
Closing this till we can reproduce it

**Comment by vivekverma2312** [comment]
Able to repro on both release and prod
<img width="1912" alt="Screenshot 2022-07-12 at 3 27 26 PM" src="https://user-images.githubusercontent.com/83569920/178469487-bd134d2f-ee61-432e-9de5-327b2f143883.png">

**Comment by rohitagarwal88** [comment]
@yaldram can you syncup with @vivekverma2312 on this issue.

**Comment by akash-codemonk** [comment]
It seems to work for me now but this is bound to happen when the image value isn't a valid url. Maybe we should just remove the image field step as it is required that it to be a valid url else it will fail which is not the case for the other text fields.

**Comment by Nikhil-Nandagopal** [comment]
@akash-codemonk yeah that makes sense. We can autobind the image and not keep it as a mandatory step in the onboarding

**Comment by kocharrahul7** [comment]
If the image URL is invalid, then it won't work even if it is auto-bound. The issue is the image value itself, not really the binding process as far as I can tell right @akash-codemonk

**Comment by akash-codemonk** [comment]
Currently the user will not be able to proceed further if the image field is not a valid url. So if we remove that step then the user can proceed further whether it is invalid or not. We autobind the image widget so that the image widget is still there and it will show the image if the image url is valid else it will show the default image. 

<img width="813" alt="test" src="https://user-images.githubusercontent.com/67054171/178649389-90b640f6-ff47-4c17-b647-c3c4b7c72f20.png">

**Comment by kocharrahul7** [comment]
Okay, got it. can we fix this on Priority

**Comment by Richarex** [comment]
This is resolved, closing it off.
![image](https://user-images.githubusercontent.com/93259714/179170158-7ffeae1e-715a-4994-8519-fad2d9673df5.png)
